api/notd/twitter_manager.py

```diff
@@ -120,6 +128,46 @@ async def refresh_twitter_credentials(self, twitterId: str) -> TwitterCredential
             await self.saver.update_twitter_credential(twitterCredentialId=twitterCredential.twitterCredentialId, accessToken=twitterCredential.accessToken, refreshToken=twitterCredential.refreshToken, expiryDate=twitterCredential.expiryDate)
         return twitterCredential
 
+    async def update_all_twitter_users_deferred(self) -> None:
+        await self.workQueue.send_message(message=UpdateAllTwitterUsersMessageContent().to_message())
+
+    async def update_all_twitter_users(self) -> None:
+        query = (
+            TwitterProfilesTable.select()
+            .with_only_columns([TwitterProfilesTable.c.twitterId])
+        )
```

Comment:
theres something interesting here - you get all the profiles here but then have to get them again (one by one) on line 166 below. might be better to just get the full profiles here and make a map below to save some queries